.github/workflows/publish-technical-documentation-release.yml

```diff
@@ -17,13 +21,16 @@ jobs:
       id-token: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-      - uses: grafana/writers-toolkit/publish-technical-documentation-release@publish-technical-documentation-release/v1
+          persist-credentials: false
+      - uses: grafana/writers-toolkit/publish-technical-documentation-release@8cc658b604c6e05c275af30163a1c7728dfe19b2 # publish-technical-documentation-release/v2
+        # This internal action's version is left as a tag instead of a pinned hash because renovate
+        # does not support this type of tag/version format without custom configuration.
```

Comment:
```suggestion
```

This comment seems outdated, possibly it's outdated on the `main` branch too so feel free to ignore this suggetion.

Renovate supports slash-separated tags now (https://github.com/renovatebot/renovate/pull/35431) and the action is actually pinned to a hash.

Reply:
Ha! Yep you’re totally right. Thanks for catching that. I think we updated to pin the hash now that you got the tag format supported in renovate, but never removed the see explanatory comments.